websites/M/MKPC/presence.ts

```diff
@@ -0,0 +1,70 @@
+const presence = new Presence({
+  clientId: "812176837748457483"
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+var user: any;
+var title: any;
+var replace: any;
+var search: any;
+var post: any;
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  };
+
+  if (document.location.pathname == "/") {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Viewing home page";
+  }
+  else if (document.location.pathname == "/forum.php") {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Viewing the forum's menu";
+    presenceData.smallImageKey = "search";
+  } else if (document.location.pathname == "index.php") {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Viewing home page";
+  } else if (document.location.pathname.includes("topic.php?topic")) {
+    presenceData.startTimestamp = browsingStamp;
+   presenceData.details = "Browsing Forum topic";
+   presenceData.smallImageKey = "search"
+  } else if (document.location.pathname.includes("profil.php?")) {    // not spelling mistake, the document location name is profil not profile. 
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Viewing user profile";
+    presenceData.smallImageKey = "search";
```

Comment:
I'm 100% sure that you can get topic title or nickname on profile and if it is possible (100% is), add privacy setting

Reply:
🤣 i tried it using the html class and not possible, but sent a message to the website dev :)

Reply:
actually found out how